aii-pxelinux/src/main/perl/pxelinux.pm

```diff
@@ -512,12 +577,16 @@ foreach my $i (qw(configure boot rescue livecd firmware install)) {
     *{$name} = sub {
         my ($self, $cfg) = @_;
 
-        pxeprint($cfg) if ($i eq 'configure');
+       foreach my $variant (PXE_VARIANT_PXELINUX, PXE_VARIANT_GRUB2) {
+            if ( $VARIANT_PARAMS[$variant]->{enabled} ) {
```

Comment:
wait, this is not set per node? so Configure configures all enabled variants for each node? (not a big issue for me, i'm just a bit surprised.)
in that case, the status should be improved somehow; otherwise this will get confusing very soon.